src/stores/utilities/run.js

```diff
@@ -188,16 +188,13 @@ const hasPlateAttributes = ({ sequencing_kit_box_barcode, wells_attributes }) =>
  * @param {smrtLinkVersion} - The SMRT Link Version of the run
  * @returns {run} - A run object
  */
-const SMRT_LINK_VERSION_V12_SEQUEL_IIE = 'v12_sequel_iie'
 const SMRT_LINK_VERSION_V12_REVIO = 'v12_revio'
```

Comment:
Is this only used in 1 place. Sometimes wonder if we need a constant for single usage ...

Reply:
Yes, it's only used in one place.  It's right, no need to define a constant if only used in one place( Perhaps it makes it more clear than directly used in function). I will remove it

Reply:
Also, with only one string it can better be changed to if (smrtLinkVersion.name === 'v12_revio')

Reply:
One other question, I think it makes more sense to pass instrumentType as we are  linking dna_control_barcode field to to the system rather than the SMRT Link Version  and it matches the comparison on UI side. What do you think?

Reply:
Makes sense.